### PR TITLE
update missing entries for 1.17.19 changelog

### PR DESCRIPTION
**Pull request by knyghtmare**
- [x] Rebels/Elves
- [x] Undead
- [x] Drakes
- [x] Dunefolk
- [x] Knalgan Alliance
- [x] Northerners
- [x] Loyalists 

Left out movement point changes of `Elvish Outrider` and `Saurian Soothsayer` as that PR has not been merged yet.

### CODE REVIEW

**Comment by stevecotton** [comment]
Feels a bit too detailed for a changelog, IMO.

File: changelog.md [hunk 1]
```diff
@@ -22,6 +22,171 @@
      * S5: Make the loss of Hans a lose-condition (#7750).
  ### Translations
    * Updated translations: British English, Czech, Finnish, French, Portuguese (Brazil)
+ ### Units
+   * Rebels:
+     * Elvish Fighter - xp cost changed from 40 to 37.
```
Comment (stevecotton) [inline changelog.md:27]: The move of the unit images in #7651 is important for UMC authors.

File: changelog.md [hunk 2]
```diff
@@ -22,6 +22,171 @@
...
+     * Cavalier - cost changed from 52 to 62.
+     * Master Bowman - cost changed from 46 to 50.
+     * Master at Arms - melee damage increased from 7 to 8, cost changed from 44 to 65.
+     * Merman Hoplite - cost changed from 43 to 45.
+     * Merman Triton - cost changed from 43 to 46.
+     * General - hp changed from 50 to 55, cost changed from 54 to 66, xp changed from 180 to 105.
+     * Great Mage - ranged damage changed from 16 to 17, cost changed from 72 to 110.
+     * Grand Marshal - hp changed from 60 to 68 cost changed from 66 to 101.
  ### Miscellaneous and Bug Fixes
    * Prevent duplicate advancements from being added to unit type. (issue #7009)
```
Comment (stevecotton) [inline changelog.md:191]: Another thing to add in Miscellaneous and Bug Fixes:
```
   * When `[unit]` recalls a unit, remove it from the recall list before firing `[event]name=unit_placed`. (issue #7769)
```

**Comment by knyghtmare** [comment]
> Feels a bit too detailed for a changelog, IMO.

If it's not detailed enough, it'd later create false positive bug reports like "Elf Champion why melee -1?"

**Comment by Pentarctagon** [comment]
There is precedent at least - https://github.com/wesnoth/wesnoth/blob/master/changelog.md#units-13

**Pull request merged**